frontend/.dockerignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+npm-debug.log
```

Comment:
Can you add at least this non extensive list to the .dockerignore as we talk before?
Dockerfile
.dockerignore
.gitignore
README.md
build
.git
.npm
.cache
**/*.class
**/.DS_Store
.vscode
logs
*.log
node_modules
npm-debug.log*